editor/src/clj/editor/gui.clj

```diff
@@ -771,17 +779,11 @@
   (property y-anchor g/Keyword (default :yanchor-none)
             (dynamic edit-type (g/constantly (properties/->pb-choicebox Gui$NodeDesc$YAnchor))))
 
+  (output visual-base-node-msg g/Any produce-visual-base-node-msg)
   (output gui-scene g/Any (g/fnk [_node-id]
                                  (node->gui-scene _node-id)))
 
   (output gpu-texture TextureLifecycle (g/constantly nil))
-  (output aabb g/Any :cached (g/fnk [pivot size]
-                                    (let [offset-fn (partial mapv + (pivot-offset pivot size))
-                                          [min-x min-y _] (offset-fn [0 0 0])
-                                          [max-x max-y _] (offset-fn size)]
-                                      (geom/coords->aabb [min-x min-y 0]
-                                                         [max-x max-y 0]))))
-
```

Comment:
The `aabb` output was already abstract in the base node. I've pushed the implementations up in the inheritance hierarchy because the `size` property was removed from the root base node. It is only used to store manual sizes and only applies to some node types, after all.